_posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md

```diff
@@ -143,52 +143,127 @@ Participation is open to all. Each team can participate in any task. This challe
 ### ASR Challenge
 
 * Data: LibriSpeech_100 + ML-SUPERBB 1h set
-* Framework: We recommend using ESPnet for a fair comparison. Feel free to let us know your preference.
-* Evaluation metrics: Word Error Rates (WERs) on Librispeech dev/test sets and Character Error Rates (CERs) on ML-SUPERB.
-* Ranking: 
-  * Word/Character Error Rate: The primary method for ranking all systems is based on their Word/Character Error Rate. This metric measures the performance of a system in terms of the accuracy of the words recognized or generated compared to a reference. 
-  * Efficiency of discrete tokens (bitrate): In addition to WER, the efficiency of discrete tokens in the systems will also be evaluated and ranked based on bitrate.
-* Submission
-  * Submission package details:
-    1. The discrete speech units corresponding to the test sets in kaldi format.
-    2. The predicted transcription corresponding to the test sets.
-    3. A technical report in Interspeech2024 paper format (no length limit)
+* Framework: We recommend to use ESPnet for fair comparison. Feel free to let us know your preferrence.
+* Evaluation metrics: 1) Character Error Rates (CERs) on LibriSpeech and ML-SUPERB evaluation sets; 2) the bitrate of the discrete unit.
+  * Character Error Rate (CER): This metric measures the performance of a system in terms of the accuracy of the words recognized or generated compared to a reference. All systems are ranked based on the CERs of the evaluation sets separately: 1) EN: dev-clean / dev-other / test-clean / test-other; 2) ML: test-1h. Note that the ranking of the EN case is based on the aggreggated CER: (total errors of {dev,test}-{clean,other}) / (total length of {dev,test}-{clean,other}) 
```

Comment:
"aggreggated" is ambiguous.
Please mention that it is a micro average of all Librispeech test sets.

Reply:
Fixed in the new PR